.gitignore

```diff
@@ -38,3 +38,6 @@ Thumbs.db
 
 ## Text Editors
 .vscode
+
+## Spellcheck script
+spellcheck.lua
```

Comment:
Was this being added to the tracked files?  I'm curious if I need to make changes to rootstock.

Reply:
I had to stick it in the root of my repo for local testing. I couldn't figure out how to get it to work anywhere else.

Reply:
(maybe the intention is not to be able to use this locally, but with it in the repo route and aspell installed then `SPELLCHECK=true bash build/build.sh` let me run things locally.

Reply:
You're right that `spellcheck.lua` currently has to be in the root of the repo because that is where were call `bash build/build.sh`.

Running locally makes sense if you have aspell..